seshdash/static/sesh-ui/js/time_series_graph.js

```diff
@@ -0,0 +1,118 @@
+function getCookie(name) {
```

Comment:
This code is copy and pasted from sesh-dashboard.js

There shoudn't be any need to copy and paste this everytime. Code is free to copy so you can put it in a seperate JS file and import it where you need it.